packages/ui/src/app/pages/Settings/EmailNotifications.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect, EventHandler } from 'react'
+
+import { PageHeader } from '@/app/components/PageHeader'
+import { PageLayout } from '@/app/components/PageLayout'
+import { SettingsTabs } from './components/SettingsTabs'
+import { EmptyPagePlaceholder } from '@/common/components/EmptyPagePlaceholder/EmptyPagePlaceholder'
+import { TransactionButton } from '@/common/components/buttons/TransactionButton'
+
+import { useMyMemberships } from '@/memberships/hooks/useMyMemberships'
+import { ColumnGapBlock, MainPanel, RowGapBlock } from '@/common/components/page/PageContent'
+import { InputComponent, InputText, ToggleCheckbox } from '@/common/components/forms'
+import { TextBig, TextMedium } from '@/common/components/typography'
+import { useToggle } from '@/common/hooks/useToggle'
+import { InformationBanner, InputEmailState } from './components/InformationBanner'
+import { SaveChangesButton } from './components/SaveChangesButton'
+
+export const EmailNotifications = () => {
+  const { email, verified, notifyset } = { email: 'test@email.com', verified: true, notifyset: true } //mocking states
+
+  const { hasMembers } = useMyMemberships()
+
+  const [subscribed, setSubscribed] = useState(!!email)
```

Comment:
shouldn't it be, like this?
`  const [subscribed, setSubscribed] = useState(!email)`
Otherwise I'm not getting the empty page.

Reply:
When the email field is empty or null then the empty page is shown .
If you set the initial email state for mocking to null then you'd get empty page.
email be queried to the backend.